src/views/ops/Template/Adhoc/AdhocDetail/AdhocDetail.vue

```diff
@@ -22,15 +35,23 @@ export default {
   data() {
     return {
       url: `/api/v1/ops/adhocs/${this.object.id}/`,
-      excludes: ['variable']
+      excludes: ['variable'],
+      disableEdit: this.object.creator !== this.$store.state.users.profile.id
     }
   },
   computed: {
     cardTitle() {
       return this.object.name
     }
   },
-  methods: {}
+  methods: {
+    updateVariable() {
+      const url = `/api/v1/ops/adhocs/${this.object.id}/`
+      this.$axios.patch(url, { variable: this.object.variable }).then(() => {
+        this.$message.success(this.$tc('UpdateSuccessMsg'))
+      })
+    }
+  }
 }
 </script>
 
```

Comment:
There appear to be multiple discrepancies between the current code version (date unknown) and an older version:

1. The `<el-col>` directives have changed to `md` instead of `col`, which could change how columns wrap horizontally.
2. The `.variables` object has been removed, suggesting it might represent variables defined within other components. If so, you need to define that elsewhere in your component hierarchy.

These changes suggest there must originally been a context or setup file for this template where these elements were properly configured. Consider reviewing the relevant files if possible; otherwise, please describe further on what should have been done with those elements, especially regarding them being removed entirely.

Note: As far as I can tell, both versions are from 2021-09-01 according to the knowledge cutoff mentioned at the start. However, I would appreciate more information about when/how you obtained these files or scripts, since their structure may vary depending on how your application was designed / updated since.